bucket/i2p-portable.json

```diff
@@ -0,0 +1,67 @@
+{
+    "version": "0.9.41",
+    "description": "An anonymizing network.",
+    "homepage": "https://geti2p.net",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://github.com/i2p/i2p.i2p/blob/master/LICENSE.txt"
+    },
+    "suggest": {
+        "JDK": [
+            "java/oraclejdk",
+            "java/openjdk"
+        ]
+    },
+    "url": "https://launchpad.net/i2p/trunk/0.9.41/%2Bdownload/i2pinstall_0.9.41_windows.exe#/dl.7z",
+    "hash": "e8d896e94696c3cb3be20cb28ea902595f82f7028622931a8ab6f451e4a3d6b5",
+    "bin": [
+        "I2P.exe",
+        "I2Psvc.exe"
+    ],
+    "shortcuts": [
+        [
+            "I2P.exe",
+            "I2P\\Start I2P (no window)",
+            "",
+            "docs\\start.ico"
+        ],
+        [
+            "I2Psvc.exe",
+            "I2P\\Start I2P (restartable)",
+            "-c \"$dir\\wrapper.config\"",
+            "docs\\start.ico"
+        ]
+    ],
+    "persist": "i2p",
+    "installer": {
+        "script": [
+            "$temp = \"$env:TEMP\\I2P\"",
+            "New-Item $temp -ItemType 'Directory' -Force | Out-Null",
+            "Move-Item \"$dir\\install.jar\" \"$temp\\install.jar\" -Force",
+            "New-Item -Path $temp -Name 'install.defaults' -Value \"INSTALL_PATH=$($dir.toString().Replace('\\', '/'))\" -Force | Out-Null",
+            "& (scoop which java) -jar \"$temp\\install.jar\" -auto | Out-Null",
+            "Remove-Item $temp -Force -Recurse",
+            "$content = @(",
+            "    'wrapper.java.additional.3=-Di2p.dir.pid=./i2p'",
+            "    'wrapper.java.additional.4=-Di2p.dir.temp=./i2p'",
+            "    'wrapper.java.additional.5=-Di2p.dir.config=./i2p'",
+            "    'wrapper.logfile=./i2p/wrapper.log'",
+            "    'wrapper.java.pidfile=./i2p/routerjvm.pid'",
+            "    'wrapper.pidfile=./i2p/i2p.pid'",
+            ")",
+            "Add-Content \"$dir\\wrapper.config\" $content -Encoding UTF8"
+        ]
+    },
+    "checkver": {
+        "url": "https://launchpad.net/i2p/trunk",
+        "regex": "Latest version is ([\\d.]+)"
+    },
+    "autoupdate": {
+        "url": "https://launchpad.net/i2p/trunk/$version/%2Bdownload/i2pinstall_$version_windows.exe#/dl.7z",
+        "hash": {
+            "url": "https://geti2p.net/download",
+            "regex": "(?sm)$basename.*?$sha256"
+        }
+    },
+    "notes": "The wrapper is NOT run (NOT portable) if you start I2P with the 'no window' icon on Windows."
```

Comment:
This does not make sense at all

> The wrapper is NOT run (NOT portable)

I guess you wanted to tell `wrapper is not run in portable mode when you strat it from no window icon`

Reply:
The warning comes from the warpper.config
```
# WARNING - The wrapper is NOT run (and this file is not used)
# if you start I2P with the 'no window' icon on Windows, or
# with the runplain.sh script on Linux. Use the 'restartable'
# icon on Windows or the i2prouter script on Linux to run the wrapper.
```

Reply:
```suggestion
    "notes": "The wrapper is not executed in portable mode if you start I2P with the 'no window' icon on Windows."
```

Reply:
Thanks!

Reply:
https://examples.yourdictionary.com/examples-of-double-negatives.html

Basically:

`wrapper is not run (not portable)` == `wrapper is runned as portable`